be/src/vec/functions/function_string.h

```diff
@@ -540,6 +545,11 @@ class FunctionStringConcatWs : public IFunction {
             } else {
                 null_list[i] = &const_null_map->get_data();
             }
+
+            if (check_column<ColumnArray>(*argument_columns[i].get())) {
```

Comment:
```suggestion
            if (check_column<ColumnArray>(argument_columns[i].get())) {
```

check_column support `const IColumn* column` as input argument.